fix-install.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+
+set -e
+set -u
+
+cd $cur__lib/bucklescript
+tar xvf libocaml.tar.gz
+mv others/* .
+mv runtime/* .
+mv stdlib-412/stdlib_modules/* .
+mv stdlib-412/* .
+
+cd $cur__lib
+ln -s $cur__target_dir/default/lib/js .
+ln -s $cur__target_dir/default/lib/es6 .
+
+ln -s $cur__bin/bsc.exe $cur__bin/bsc
```

Comment:
feel free to address this in a subsequent PR, but I think we could just fix this in the dune files

```lisp
(install
  (files (bsc.exe as bsc))
  (section bin))
```

Reply:
thank you, this works